backend/serverless.yml

```diff
@@ -82,13 +82,13 @@ functions:
     handler: handler.viewPins
     events:
       - http:
-          path: /dev/yourPins/pinsList
+          path: /yourPins/pinsList
           method: GET
   viewCategories:
     handler: handler.viewCategories
     events:
       - http:
-          path: /dev/yourPins/categories
+          path: /yourPins/categories
```

Comment:
For this one, and the one above, by putting `/dev` here, when you make a request, you would have had to use `/dev/dev/yourPins/categories` which is not what you meant. You can omit that part of the endpoint in your serverless.yml & in your handler code.

AWS API Gateway has the concept of "stages" to allow you to deploy different versions of your code to different paths so that they dont interfere with each other. For example, you can have a known working version of your code be deployed to the "prod" stage (i.e. `/prod/yourPins/categories`), whereas your work-in-progress code can be deployed to the "dev" stage (i.e. `/dev/yourPins/categories`). Serverless uses the stage name as a prefix for your urls, and it uses "dev" as the default stage name (L29 in serverless.yml). 

We're not using stages as part of this class which is why we're using `/dev` as the path prefix all over the frontend code. It is a useful feature though if you're developing as part of a larger team. I thought I'd share in case you were confused as to why /dev is part of the URL.